tests/test_scylla_repository.py

```diff
@@ -16,4 +16,14 @@ def test_setup_release_oss(self):
     @patch.dict('os.environ', {'SCYLLA_PRODUCT': 'scylla-enterprise'})
     def test_setup_release_enterprise(self):
         cdir, version = setup(version="release:2020.1.5", verbose=True)
-        assert version == '2020.1.5'
\ No newline at end of file
+        assert version == '2020.1.5'
+
+    def test_setup_unstable_master_new_url(self):
```

Comment:
mark those new tests with
`@pytest.mark.skip("slow integration test")`

or mark this whole class, with that.

they are too long for the github actions to run them each time

Reply:
also one of those failed:
```
2021-01-20T16:43:08.8474441Z =================================== FAILURES ===================================
2021-01-20T16:43:08.8475550Z ___________ TestScyllaRepository.test_setup_unstable_master_old_url ____________
2021-01-20T16:43:08.8476862Z tests/test_scylla_repository.py:29: in test_setup_unstable_master_old_url
2021-01-20T16:43:08.8478639Z     assert version == '4.4.dev'
2021-01-20T16:43:08.8479808Z E   AssertionError: assert '3.0' == '4.4.dev'
2021-01-20T16:43:08.8480854Z E     - 4.4.dev
2021-01-20T16:43:08.8481531Z E     + 3.0
2021-01-20T16:43:08.8482741Z         cdir       = '/home/runner/.ccm/scylla-repository/unstable/master/2020-08-29T22_24_05Z'
2021-01-20T16:43:08.8484313Z         self       = <tests.test_scylla_repository.TestScyllaRepository object at 0x7f43f8a4fb80>
2021-01-20T16:43:08.8485691Z         version    = '3.0'
2021-01-20T16:43:08.8486920Z ----------------------------- Captured stdout call -----------------------------
2021-01-20T16:43:08.8487852Z 3.0
2021-01-20T16:43:08.8488980Z ------------------------------ Captured log call -------------------------------
```